Index: main.cpp
===================================================================
--- main.cpp	(revision 77899)
+++ main.cpp	(working copy)
@@ -210,7 +210,14 @@
 struct wxMSWCommandLineArguments
 {
     wxMSWCommandLineArguments() { argc = 0; argv = NULL; }
-
+    ~wxMSWCommandLineArguments()
+    {
+        // See http://forums.wxwidgets.org/viewtopic.php?f=1&t=33716
+        if (argc != 0)
+        {
+            Free();
+        }
+    }
     void Init(const wxArrayString& args)
     {
         argc = args.size();
