Index: main.cpp
===================================================================
--- main.cpp	(revision 74976)
+++ main.cpp	(working copy)
@@ -306,7 +306,14 @@
 struct wxMSWCommandLineArguments
 {
     wxMSWCommandLineArguments() { argc = 0; argv = NULL; }
-
+    ~wxMSWCommandLineArguments()
+    {
+        // See http://forums.wxwidgets.org/viewtopic.php?f=1&t=33716
+        if (argc != 0)
+        {
+            Free();
+        }
+    }
     void Init(const wxArrayString& args)
     {
         argc = args.size();
@@ -354,7 +361,6 @@
         return false;
 #endif // NEED_UNICODE_CHECK
 
-
     // remember the parameters Windows gave us
     wxSetInstance(hInstance);
 #ifdef __WXMSW__
@@ -435,4 +441,4 @@
     wxhInstance = hInst;
 }
 
-#endif // wxUSE_BASE
+#endif // wxUSE_BASE
\ No newline at end of file
